packages/config-transformer/tests/Converter/ConfigFormatConverter/YamlToPhp/Fixture/normal/workflow-constant.yaml

```diff
@@ -0,0 +1,14 @@
+transitions:
+    !php/const App\Entity\Plant::TRANSITION_PLANT:
+        from: seed
+        to:   planted
+-----
```

Comment:
There seem unrelated array following.
Would this also work?

```suggestion
transitions:
    !php/const App\Entity\Plant::TRANSITION_PLANT: some_value
-----
```

Reply:
I'm not sure, but obviously this needs to work too.